about.html

```diff
@@ -0,0 +1,443 @@
+                    />
+                  </a>
+                </div>
+              </div>
+            </div>
+          </div>
+          <!-- Second row -->
+          <div class="flex justify-center md:col-span-5 mt-6">
+            <!-- Fourth card -->
+            <div
+              class="w-full max-w-sm bg-white border border-gray-200 rounded-lg shadow dark:bg-gray-800 dark:border-gray-700"
+            >
+              <div class="p-6">
+                <!-- Card content -->
+                <img
+                  class="w-24 h-24 mb-3 rounded-full shadow-lg mx-auto"
+                  src="assets/images/cara.svg"
+                  alt="Profile picture"
+                />
+                <h5
+                  class="mb-1 text-xl font-medium text-gray-900 dark:text-white text-center"
+                >
+                  Cara
+                </h5>
+                <span
+                  class="text-sm text-gray-500 dark:text-gray-400 text-center block"
+                  >CyberOps Apprentice</span
+                >
+                <div class="flex justify-center mt-4">
+                  <a
+                    href="https://github.com/yourusername"
+                    target="_blank"
+                    rel="noopener noreferrer"
+                  >
+                    <img
+                      src="assets/icons/github.svg"
+                      alt="GitHub Icon"
+                      class="w-8 h-8"
+                    />
+                  </a>
+                </div>
+              </div>
+            </div>
+            <!-- Fifth card -->
+            <div
+              class="w-full max-w-sm bg-white border border-gray-200 rounded-lg shadow dark:bg-gray-800 dark:border-gray-700 mt-6 md:mt-0"
+            >
+              <div class="p-6">
+                <!-- Card content -->
+                <img
+                  class="w-24 h-24 mb-3 rounded-full shadow-lg mx-auto"
+                  src="assets/images/devina.svg"
+                  alt="Profile picture"
+                />
+                <h5
+                  class="mb-1 text-xl font-medium text-gray-900 dark:text-white text-center"
+                >
+                  Devina
+                </h5>
+                <span
+                  class="text-sm text-gray-500 dark:text-gray-400 text-center block"
+                  >CyberOps Apprentice</span
+                >
+                <div class="flex justify-center mt-4">
+                  <a
+                    href="https://github.com/yourusername"
+                    target="_blank"
+                    rel="noopener noreferrer"
+                  >
+                    <img
+                      src="assets/icons/github.svg"
+                      alt="GitHub Icon"
+                      class="w-8 h-8"
+                    />
+                  </a>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </main>
```

Comment:
Responsiveness needs changing for second row on tablet/mobile. On tablet one card should shift up to complete first row to have 3 rows (2-2-1). On mobile needs changing so fifth card moves below 4th card, currently cards stick together. 
Also missing links for github, currently all set to /yourusername

Reply:
kept as desktop view for now until all pages are complete to then look at responsiveness, github usernames added